code/modules/projectiles/boxes_magazines/external/pistol.dm

```diff
@@ -83,3 +83,13 @@
 	caliber = ".50"
 	max_ammo = 7
 	multiple_sprites = AMMO_BOX_PER_BULLET
+
+/obj/item/ammo_box/magazine/liberator10mm

```

Comment:
you know this will pre-load it with bullets right?

Reply:
I would tell you to set var/start_empty = TRUE but I have no idea where you expect people to be getting their 9mm bullets from

Reply:
Most likely from the hacked autolathe. If they dont have access to this, they get the 5 bullets so that the weapon isnt completely useless upon creation.